specification/attributes/column_naming_convention.md

```diff
@@ -0,0 +1,34 @@
+# Column naming convention
+
+Column IDs provided in billing datasets following a consistent naming convention reduces friction for FinOps
+practitioners that consume the datasets for analysis, reporting, and other use cases.
+
+All columns defined in the FOCUS specification MUST follow the naming requirements listed below. Provider generated
+columns SHOULD adopt these same naming requirements over time.
+
+## Attribute ID
+
+ColumnNamingConvention
+
+## Attribute name
+
+Column Naming Convention
+
+## Description
+
+Naming Convention for columns appearing in billing data.
+
+## Requirements
+
+* Column IDs MUST use Pascal case
```

Comment:
nit: Wikipedia refers to Pascal case as "upper camel case". Do we have an opinion on which we should use? I prefer Pascal case, but just calling it out to get others' perspective. Please resolve if no other comments 🙂

Reply:
We should just link to an online explanation of the format. I also like Pascal case.

Reply:
Same I like Pascal case...but wow its origins are back to the Pascal language...was hoping to find a quick document on the format but haven't found anything particularly useful.

Reply:
Seems like this: https://techterms.com/definition/pascalcase is a decent one to link. Adding that.